scenario/GovernanceScenario.ts

```diff
@@ -143,7 +143,68 @@ scenario('add new asset',
 
     // Try to supply new token and borrow base
     const baseAssetAddress = await comet.baseToken();
-    const borrowAmount = 1_000n * (await comet.baseScale()).toBigInt();
+    const borrowAmount = 1000n * (await comet.baseScale()).toBigInt();
+    await dogecoin.connect(albert.signer).approve(comet.address, exp(100, 8));
+    await albert.supplyAsset({ asset: dogecoin.address, amount: exp(100, 8) });
+    await albert.withdrawAsset({ asset: baseAssetAddress, amount: borrowAmount });
+
+    expect(await albert.getCometCollateralBalance(dogecoin.address)).to.be.equal(exp(100, 8));
+    expect(await albert.getCometBaseBalance()).to.be.equal(-borrowAmount);
+  });
+
+scenario('add new asset (mainnet-weth)',
```

Comment:
Curious why we need to separate out this scenario? Shouldn't it just work for both deployments?

Reply:
the original test attempts to borrow 1,000 units of the base asset using 100 assets of DOGE (with a price of $1,000 per DOGE).

that works with a USDC base, since a $1,000 borrow can be collateralized with $100,000 worth of DOGE.

but borrowing 1,000 ETH costs about $1.5M, so the borrow is uncollateralized

Reply:
Ah I see, that is indeed annoying. Another approach could be to set the borrow amount to be the `borrowBaseMin` instead of a constant. But this works too.

Reply:
Shouldn't we just have a PriceConstraint?

Reply:
that's a clever idea. So you'd specify a price for the base asset and/or the collateral assets

we're having a similar conversation on the [Polygon PR](https://github.com/compound-finance/comet/pull/598#discussion_r1013496694).

Another approach would be to write a helper function returned the amount of a given asset required to cover a borrow of X base units

Reply:
Haha, yep just commented there, we can pick one ;) But I think its more desirable to use literal numbers in properties where we can (👨‍🍳 has long advocated for this and I kind of agree)

Reply:
#615